app/invocation/invocation_model.tsx

```diff
@@ -500,4 +500,12 @@ export default class InvocationModel {
   isQuery() {
     return this.getCommand() == "query";
   }
+
+  hasChunkedEventLogs(): boolean {
+    return this.invocations[0]?.hasChunkedEventLogs || false;
+  }
+
+  isComplete(): boolean {
```

Comment:
nit: Are we using this somewhere? It appears vestigial.

Reply:
Good catch - done